src/views/InteractionDesigner.vue

```diff
@@ -432,35 +432,35 @@ export default {
 
   .tree-sidebar-container {
     .tree-sidebar {
-      &.category-0 {
-        border-color: $category-0-light;
-        background-color: $category-0-faint;
-        border-radius: 0.3em;
-
-        h3 {
-          color: $category-0-dark;
-        }
-      }
-
-      &.category-1 {
-        border-color: $category-1-light;
-        background-color: $category-1-faint;
-        border-radius: 0.3em;
-
-        h3 {
-          color: $category-1-dark;
-        }
-      }
-
-      &.category-2 {
-        border-color: $category-2-light;
-        background-color: $category-2-faint;
-        border-radius: 0.3em;
-
-        h3 {
-          color: $category-2-dark;
-        }
-      }
+      //  &.category-0 {
+      //    border-color: $category-0-light;
+      //    background-color: $category-0-faint;
+      //    border-radius: 0.3em;
+      //
+      //    h3 {
+      //      color: $category-0-dark;
+      //    }
+      //  }
```

Comment:
Based on our discussion today to remove the block outline color. Can we remove this commented code?

Reply:
Ah! I'm a big fan of removing dead, stale and commented-out code. We're in agreement, I'll remove it 🙆‍♀️